sympy/stats/joint_rv_types.py

```diff
@@ -70,35 +74,36 @@ def JointRV(symbol, pdf, _set=None):
         return JointPSpace(symbol, dist).value
     return jrv
 
-#-------------------------------------------------------------------------------
+
+# -------------------------------------------------------------------------------
 # Multivariate Normal distribution ---------------------------------------------------------
 
 class MultivariateNormalDistribution(JointDistribution):
     _argnames = ['mu', 'sigma']
 
-    is_Continuous=True
+    is_Continuous = True
 
     @property
     def set(self):
         k = len(self.mu)
-        return S.Reals**k
+        return S.Reals ** k
 
     @staticmethod
     def check(mu, sigma):
         _value_check(len(mu) == len(sigma.col(0)),
-            "Size of the mean vector and covariance matrix are incorrect.")
-        #check if covariance matrix is positive definite or not.
+                     "Size of the mean vector and covariance matrix are incorrect.")
+        # check if covariance matrix is positive definite or not.
         _value_check((i > 0 for i in sigma.eigenvals().keys()),
-            "The covariance matrix must be positive definite. ")
+                     "The covariance matrix must be positive definite. ")
 
     def pdf(self, *args):
         mu, sigma = self.mu, self.sigma
         k = len(mu)
         args = ImmutableMatrix(args)
         x = args - mu
-        return  S(1)/sqrt((2*pi)**(k)*det(sigma))*exp(
-            -S(1)/2*x.transpose()*(sigma.inv()*\
-                x))[0]
+        return S(1) / sqrt((2 * pi) ** (k) * det(sigma)) * exp(
+            -S(1) / 2 * x.transpose() * (sigma.inv() *
+                                         x))[0]
```

Comment:
if you want to change the style, maybe you should open a new PR. This edit is useless.